adapter_docs/training.md

```diff
@@ -13,14 +13,14 @@ pip install -r ./examples/pytorch/<your_examples_folder>/requirements.txt
 
 ## Train a Task Adapter
 
-Training a task adapter module on a dataset only requires minor modifications from training the full model.
+Training a task adapter module on a dataset only requires minor modifications from training a whole model.
```

Comment:
```suggestion
Training a task adapter module on a dataset only requires minor modifications compared to training the entire model.
```
For consistency, I would use "entire model" instead of "whole model" as you have used "entire model" everywhere else.